config/checkstyle-examples-checks.xml

```diff
@@ -115,6 +115,8 @@
       <property name="format" value="^class Example\d+"/>
       <property name="minimum" value="1"/>
       <property name="maximum" value="1"/>
+      <message key="regexp.minimum"
+               value="Example test input class name must match ''Example\d+''"/>
```

Comment:
Before:
```
[ERROR] [checkstyle] [ERROR] /home/nick/IdeaProjects/checkstyle/src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/design/finalclass/Example1.java:1: File does not contain at least 1 matches for pattern '^class Example\d+'. [exampleClassName]
```

After:
```
[ERROR] [checkstyle] [ERROR] /home/nick/IdeaProjects/checkstyle/src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/design/finalclass/Example1.java:1: Example test input class name must match 'Example\d+' [exampleClassName]

```

It was not obvious what the original message meant, this needs to be more friendly for contributors.

Reply:
Agreed.